llvm/tools/llvm-profdata/llvm-profdata.cpp

```diff
@@ -130,6 +131,12 @@ cl::opt<std::string>
                    cl::desc("For merge, use the provided unstripped bianry to "
                             "correlate the raw profile."),
                    cl::sub(MergeSubcommand));
+cl::list<std::string> DebugFileDirectory(
+    "debug-file-directory",
```

Comment:
We already have a flag spelled `debug-info`, but this flag is unrelated.

Reply:
We already have the same option in other tools like [`llvm-cov`](https://github.com/llvm/llvm-project/blob/d8bffa9018c88ef6ce441bb44d7b7d7a9091e583/llvm/tools/llvm-cov/CodeCoverage.cpp#L684) and [`llvm-symbolizer`](https://github.com/llvm/llvm-project/blob/d8bffa9018c88ef6ce441bb44d7b7d7a9091e583/llvm/tools/llvm-symbolizer/Opts.td#L28) and I think it's worth being consistent in option spelling across tools.

Reply:
I used the `debug-file-directory` flag to be consistent with all the other tools that supports`debuginfod`. Besides the ones that @petrhosek listed, we also have [llvm-objdump](https://github.com/llvm/llvm-project/blob/54194e1506bdd6dc37988678a8047ad4d48168fa/llvm/docs/CommandGuide/llvm-objdump.rst?plain=1#L137).